collector/exec_bsd.go

```diff
@@ -72,6 +75,12 @@ func NewExecCollector() (Collector, error) {
 				description: "Number of fork() calls since system boot.  Resets at architeture unsigned integer.",
 				mib:         "vm.stats.vm.v_forks",
 			},
+			{
+				name:        "boot_time",
```

Comment:
Or perhaps this isn't under `exec` at all?  I've got this to be node_exec_node_boot_timestamp_seconds, "makes sense to me".